### ISSUE TITLE
Raise and lower brushes not working

### ISSUE THREAD

**Issue by MegaNarwhal** [SBPrime/AsyncWorldEdit#337]
Using AWE 3.7.5 with WorldEdit 7.2.0-beta-02 on Java 11, when trying to use the raise or lower brushes, no blocks are changed and I get this in the console:
```
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] Exception: java.lang.reflect.InaccessibleObjectException, Unable to make field private final java.lang.String java.lang.module.ModuleDescriptor.name accessible: module java.base does not "opens java.lang.module" to unnamed module @75ac0ef8
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:340)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:280)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:176)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] java.base/java.lang.reflect.Field.setAccessible(Field.java:170)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.primesoft.asyncworldedit.asyncinjector.scanner.ClassScanner.unpack(ClassScanner.java:311)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.primesoft.asyncworldedit.asyncinjector.scanner.ClassScanner.scan(ClassScanner.java:188)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.primesoft.asyncworldedit.asyncinjector.async.AsyncOperationProcessor.process(AsyncOperationProcessor.java:132)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.primesoft.asyncworldedit.injector.core.visitors.Helpers.executeMethodEx(Helpers.java:96)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.function.operation.Operations.completeLegacy(Operations.java)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.command.tool.brush.OperationFactoryBrush.build(OperationFactoryBrush.java:57)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.command.tool.BrushTool.actPrimary(BrushTool.java:209)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.primesoft.asyncworldedit.worldedit.command.tool.WrappedBrushTool.actPrimary(WrappedBrushTool.java:146)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.extension.platform.PlatformManager.handlePlayerInput(PlatformManager.java:390)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] jdk.internal.reflect.GeneratedMethodAccessor31.invoke(Unknown Source)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] java.base/java.lang.reflect.Method.invoke(Method.java:566)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.util.eventbus.EventBus.nonwrapped_dispatch(EventBus.java:193)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.primesoft.asyncworldedit.worldedit.util.eventbus.EventBusWrapper.dispatch(EventBusWrapper.java:107)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.util.eventbus.EventBus.nonwrapped_post(EventBus.java:181)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.primesoft.asyncworldedit.worldedit.util.eventbus.EventBusWrapper.post(EventBusWrapper.java:71)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.primesoft.asyncworldedit.worldedit.CancelabeEditSession$EventBusWrapESEvent.post(CancelabeEditSession.java:520)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.WorldEdit.handleRightClick(WorldEdit.java:608)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.sk89q.worldedit.bukkit.WorldEditListener.onPlayerInteract(WorldEditListener.java:142)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor18.execute(Unknown Source)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:498)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:465)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:461)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.PlayerConnection.a(PlayerConnection.java:1505)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:28)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:5)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:23)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.TickTask.run(SourceFile:18)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.MinecraftServer.aZ(MinecraftServer.java:1136)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.MinecraftServer.executeNext(MinecraftServer.java:1129)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.MinecraftServer.sleepForTick(MinecraftServer.java:1090)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1004)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] java.base/java.lang.Thread.run(Thread.java:834)
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] Class: class java.lang.module.ModuleDescriptor
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] Object: module { name: java.base@11.0.8, uses: [javax.security.auth.spi.LoginModule, jdk.internal.logger.DefaultLoggerFinder, java.nio.channels.spi.SelectorProvider, java.util.spi.CurrencyNameProvider, java.text.spi.DecimalFormatSymbolsProvider, java.util.spi.TimeZoneNameProvider, java.nio.file.spi.FileTypeDetector, java.text.spi.NumberFormatProvider, java.net.ContentHandlerFactory, java.util.spi.ToolProvider, sun.util.locale.provider.LocaleDataMetaInfo, java.text.spi.DateFormatSymbolsProvider, java.time.chrono.Chronology, java.nio.channels.spi.AsynchronousChannelProvider, sun.text.spi.JavaTimeDateTimePatternProvider, java.security.Provider, sun.util.resources.LocaleData$CommonResourceBundleProvider, java.util.spi.CalendarDataProvider, java.text.spi.BreakIteratorProvider, java.time.chrono.AbstractChronology, sun.util.resources.LocaleData$SupplementaryResourceBundleProvider, java.nio.file.spi.FileSystemProvider, java.net.spi.URLStreamHandlerProvider, java.text.spi.CollatorProvider, java.util.spi.ResourceBundleProvider, java.lang.System$LoggerFinder, sun.util.spi.CalendarProvider, java.nio.charset.spi.CharsetProvider, java.time.zone.ZoneRulesProvider, java.text.spi.DateFormatProvider, java.util.spi.LocaleNameProvider, java.util.spi.CalendarNameProvider, java.util.spi.ResourceBundleControlProvider], exports: [sun.nio.ch to [java.management, jdk.net, jdk.crypto.cryptoki, jdk.sctp], java.security.spec, java.time.chrono, sun.net.ext to [jdk.net], jdk.internal.org.objectweb.asm.signature to [jdk.scripting.nashorn], java.nio.charset, sun.net.dns to [jdk.naming.dns, java.security.jgss], java.lang.module, java.util, java.nio.file.spi, javax.security.auth, java.nio.channels, jdk.internal.jimage to [jdk.jlink], javax.security.auth.x500, java.nio.file, javax.crypto.interfaces, javax.crypto.spec, sun.security.provider to [jdk.crypto.cryptoki, java.security.jgss, jdk.security.auth, java.rmi], sun.security.jca to [jdk.crypto.cryptoki, jdk.naming.dns, java.smartcardio, jdk.crypto.ec], jdk.internal.loader to [java.instrument, java.logging, java.naming], sun.reflect.generics.reflectiveObjects to [java.desktop], sun.security.rsa to [jdk.crypto.cryptoki], jdk.internal.org.xml.sax to [jdk.jfr], java.util.concurrent.locks, sun.net.util to [java.desktop, java.net.http, jdk.jconsole], java.lang.ref, jdk.internal.reflect to [java.sql, jdk.unsupported, jdk.internal.vm.ci, jdk.dynalink, jdk.scripting.nashorn, java.logging, java.sql.rowset], jdk.internal.jmod to [jdk.jlink, jdk.compiler], sun.security.tools to [jdk.jartool], java.lang, jdk.internal.module to [jdk.jlink, jdk.jfr, java.management.rmi, java.instrument, jdk.jartool], sun.security.pkcs to [jdk.crypto.ec, jdk.jartool], jdk.internal.ref to [java.desktop], java.lang.invoke, jdk.internal.util.xml to [jdk.jfr], java.util.concurrent.atomic, java.util.regex, java.security.acl, jdk.internal.perf to [jdk.internal.jvmstat, java.management, jdk.management.agent], jdk.internal.org.objectweb.asm.util to [jdk.scripting.nashorn, jdk.jfr], java.math, java.util.zip, sun.reflect.annotation to [jdk.compiler], java.lang.reflect, sun.reflect.misc to [java.datatransfer, java.management.rmi, java.desktop, java.rmi, java.management, java.sql.rowset], javax.security.auth.login, sun.security.action to [jdk.crypto.ec, java.security.jgss, java.desktop], java.nio.charset.spi, sun.security.validator to [jdk.jartool], jdk.internal.event to [jdk.jfr], sun.security.x509 to [jdk.crypto.ec, jdk.jartool, jdk.crypto.cryptoki], java.net, java.nio.file.attribute, java.security, java.security.cert, sun.security.internal.spec to [jdk.crypto.cryptoki], jdk.internal.jimage.decompressor to [jdk.jlink], java.time.format, sun.nio.cs to [jdk.charsets, java.desktop], sun.net.www.protocol.http to [java.security.jgss], sun.util.cldr to [jdk.jlink], javax.security.auth.spi, jdk.internal.access to [java.desktop, java.rmi, java.management, jdk.net, java.logging, jdk.jlink, java.naming], sun.util.logging to [java.desktop, java.logging, java.prefs], jdk.internal.org.xml.sax.helpers to [jdk.jfr], java.time.temporal, sun.security.ssl to [java.security.jgss], jdk.internal.org.objectweb.asm.tree to [jdk.jlink, jdk.jfr], java.nio.channels.spi, sun.security.util.math.intpoly to [jdk.crypto.ec], sun.security.util.math to [jdk.crypto.ec], sun.security.provider.certpath to [java.naming], sun.security.internal.interfaces to [jdk.crypto.cryptoki], java.util.concurrent, sun.net to [jdk.naming.dns, java.net.http], sun.util.locale.provider to [jdk.localedata, jdk.jlink, java.desktop], sun.util.resources to [jdk.localedata], jdk.internal.org.objectweb.asm.commons to [jdk.scripting.nashorn, jdk.jfr], jdk.internal.util.jar to [jdk.jartool], java.time.zone, jdk.internal.logger to [java.logging], javax.net.ssl, java.io, sun.security.timestamp to [jdk.jartool], jdk.internal.org.objectweb.asm to [jdk.scripting.nashorn, jdk.jlink, jdk.jfr, jdk.jartool], java.util.function, jdk.internal to [jdk.jfr], javax.security.cert, jdk.internal.vm.annotation to [jdk.internal.vm.ci, jdk.unsupported], com.sun.security.ntlm to [java.security.sasl], java.text.spi, jdk.internal.platform to [jdk.management], java.time, java.text, sun.net.www to [java.net.http, jdk.jartool, java.desktop], java.security.interfaces, javax.net, java.util.spi, javax.crypto, java.util.stream, jdk.internal.vm to [jdk.internal.jvmstat, jdk.management.agent], javax.security.auth.callback, java.lang.annotation, java.nio, jdk.internal.util.xml.impl to [jdk.jfr], sun.security.util to [jdk.naming.ldap, java.naming, java.desktop, jdk.crypto.ec, jdk.crypto.cryptoki, java.smartcardio, jdk.security.auth, jdk.security.jgss, jdk.jartool, java.xml.crypto, java.rmi, java.security.jgss, java.security.sasl], java.util.jar, java.net.spi, jdk.internal.misc to [jdk.compiler, java.management, java.xml, jdk.unsupported, jdk.scripting.nashorn.shell, jdk.scripting.nashorn, jdk.jfr, java.naming, jdk.internal.vm.ci, java.rmi, java.net.http, jdk.charsets, java.logging, jdk.attach, java.security.jgss, jdk.jshell, java.desktop]], provides: [java.nio.file.spi.FileSystemProvider with [jdk.internal.jrtfs.JrtFileSystemProvider]] }
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[14:55:36] [Server thread/INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[14:55:36] [Server thread/ERROR]: Could not dispatch event: com.sk89q.worldedit.event.platform.PlayerInputEvent@39a3bbe7 to handler EventHandler{priority=NORMAL}
java.lang.reflect.InvocationTargetException: null
	at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:75) ~[?:?]
	at com.sk89q.worldedit.util.eventbus.EventBus.nonwrapped_dispatch(EventBus.java:193) ~[?:?]
	at org.primesoft.asyncworldedit.worldedit.util.eventbus.EventBusWrapper.dispatch(EventBusWrapper.java:107) ~[?:?]
	at com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java) ~[?:?]
	at com.sk89q.worldedit.util.eventbus.EventBus.nonwrapped_post(EventBus.java:181) ~[?:?]
	at org.primesoft.asyncworldedit.worldedit.util.eventbus.EventBusWrapper.post(EventBusWrapper.java:71) ~[?:?]
	at org.primesoft.asyncworldedit.worldedit.CancelabeEditSession$EventBusWrapESEvent.post(CancelabeEditSession.java:520) ~[?:?]
	at com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java) ~[?:?]
	at com.sk89q.worldedit.WorldEdit.handleRightClick(WorldEdit.java:608) ~[?:?]
	at com.sk89q.worldedit.bukkit.WorldEditListener.onPlayerInteract(WorldEditListener.java:142) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor18.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.1.jar:git-Paper-114]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.16.1.jar:git-Paper-114]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.1.jar:git-Paper-114]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.1.jar:git-Paper-114]
	at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:498) ~[patched_1.16.1.jar:git-Paper-114]
	at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:465) ~[patched_1.16.1.jar:git-Paper-114]
	at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:461) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.PlayerConnection.a(PlayerConnection.java:1505) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:28) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:5) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:23) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.TickTask.run(SourceFile:18) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.MinecraftServer.aZ(MinecraftServer.java:1136) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.MinecraftServer.executeNext(MinecraftServer.java:1129) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.MinecraftServer.sleepForTick(MinecraftServer.java:1090) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1004) ~[patched_1.16.1.jar:git-Paper-114]
	at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-114]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.reflect.InvocationTargetException
	at jdk.internal.reflect.GeneratedMethodAccessor31.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?]
	at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?]
	... 32 more
Caused by: java.lang.ClassCastException: class java.lang.String cannot be cast to class java.lang.Integer (java.lang.String and java.lang.Integer are in module java.base of loader 'bootstrap')
	at com.sk89q.worldedit.bukkit.fastutil.ints.Int2ObjectFunction.get(Int2ObjectFunction.java:122) ~[?:?]
	at com.sk89q.worldedit.bukkit.fastutil.ints.Int2ObjectMap.get(Int2ObjectMap.java:186) ~[?:?]
	at java.util.concurrent.ConcurrentHashMap.equals(ConcurrentHashMap.java:1360) ~[?:?]
	at java.util.HashMap.getNode(HashMap.java:567) ~[?:?]
	at java.util.HashMap.containsKey(HashMap.java:591) ~[?:?]
	at java.util.HashSet.contains(HashSet.java:204) ~[?:?]
	at org.primesoft.asyncworldedit.asyncinjector.scanner.ClassScanner.scan(ClassScanner.java:178) ~[?:?]
	at org.primesoft.asyncworldedit.asyncinjector.async.AsyncOperationProcessor.process(AsyncOperationProcessor.java:132) ~[?:?]
	at org.primesoft.asyncworldedit.injector.core.visitors.Helpers.executeMethodEx(Helpers.java:96) ~[?:?]
	at com.sk89q.worldedit.function.operation.Operations.completeLegacy(Operations.java) ~[?:?]
	at com.sk89q.worldedit.command.tool.brush.OperationFactoryBrush.build(OperationFactoryBrush.java:57) ~[?:?]
	at com.sk89q.worldedit.command.tool.BrushTool.actPrimary(BrushTool.java:209) ~[?:?]
	at org.primesoft.asyncworldedit.worldedit.command.tool.WrappedBrushTool.actPrimary(WrappedBrushTool.java:146) ~[?:?]
	at com.sk89q.worldedit.extension.platform.PlatformManager.handlePlayerInput(PlatformManager.java:390) ~[?:?]
	at jdk.internal.reflect.GeneratedMethodAccessor31.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?]
	at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?]
	... 32 more
```

**Comment by SBPrime** [comment]
In awe enable debug mode do the same operation and send me the log.

**Comment by MegaNarwhal** [comment]
[latest.log](https://github.com/SBPrime/AsyncWorldEdit/files/4977775/latest.log)

**Comment by SBPrime** [comment]
Fixed in 3.7.6
